Makefile

```diff
@@ -46,14 +46,17 @@ endif
 #----------------------------------------------------------------------
 
 # Explicitly set the default target to do everything that isn’t already done
-default: | init assets all public ;
+default: | init lint assets all public ;
 
 # Run anything that needs doing post-checkout to make this buildable
 init: node_modules ;
 
 # Use building English language as a check to see if everything works
 test: en ;
 
+lint:
+	find source -type f -name '*.json' -print -exec jsonlint -q '{}' \;
```

Comment:
> AFAICT from find(1) the problems with -exec aren't a big deal here

Neither the [filename attack](https://www.gnu.org/software/findutils/manual/html_mono/find.html#Problems-with-_002dexec-and-filenames), nor the symbolic link dereferencing [race condition](https://www.gnu.org/software/findutils/manual/html_mono/find.html#Race-Conditions-with-_002dexec) are relevant here afais, so `-exec` is indeed fine.